tests/roformer/test_tokenization_roformer.py

```diff
@@ -71,3 +71,11 @@ def test_training_new_tokenizer(self):
     # can't train new_tokenizer via Tokenizers lib
     def test_training_new_tokenizer_with_special_tokens_change(self):
         pass
+
+    # can't serialise custom PreTokenizer
+    def test_save_slow_from_fast_and_reload_fast(self):
```

Comment:
This test was failing with `Exception: Custom PreTokenizer cannot be serialized`:

```
    def test_save_slow_from_fast_and_reload_fast(self):
        if not self.test_slow_tokenizer or not self.test_rust_tokenizer:
            # we need both slow and fast versions
            return
    
        for tokenizer, pretrained_name, kwargs in self.tokenizers_list:
            with self.subTest(f"{tokenizer.__class__.__name__} ({pretrained_name})"):
                with tempfile.TemporaryDirectory() as tmp_dir_1:
                    # Here we check that even if we have initialized a fast tokenizer with a tokenizer_file we can
                    # still save only the slow version and use these saved files to rebuild a tokenizer
                    tokenizer_fast_old_1 = self.rust_tokenizer_class.from_pretrained(
                        pretrained_name, **kwargs, use_fast=True
                    )
                    tokenizer_file = os.path.join(tmp_dir_1, "tokenizer.json")
>                   tokenizer_fast_old_1.backend_tokenizer.save(tokenizer_file)
E                   Exception: Custom PreTokenizer cannot be serialized
```

Looking at this similar issue https://github.com/huggingface/tokenizers/issues/613 it seems that RoFormer belong to the class of tokenizers that can't be saved with `tokenizer.save()`. 

I'm not very familiar with RoFormer, so happy to debug this further if we expect this test really should pass.

Reply:
This one I think we can definitely skip!